crates/bevy_winit/src/winit_event.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(deprecated)]
```

Comment:
These should be localized to where they're supposed to be used.

Reply:
How is this achieved for the `WinitEvent` enum in `winit_event.rs`? I've tried the following:

```rust
pub enum WinitEvent {
  // ..
  #[allow(deprecated)]
  ReceivedCharacter(ReceivedCharacter),
  // ..
}
```

and

```rust
#[allow(deprecated)]
pub enum WinitEvent
```

but clippy still seems to complain.

Reply:
I think it needs to go on the functions that use the WinitEvent